src/components/Article.jsx

```diff
@@ -0,0 +1,30 @@
+import Back from "./Back";
+
+export default function Article({
+  title,
+  description,
+  link,
+  src,
+  viewTrending,
+}) {
+  return (
+    <div className="pt-8 pb-8">
+      <Back onClick={viewTrending} />
```

Comment:
This is a bit confusing because we don't know whether it's a prop or an event. 
My suggestion is to make it an event if possible.